spec/books_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../book'
+
+describe Book do
```

Comment:
- [ Required ] I  think it would be better if you add a test case to test the `add_rental` method in the `Book` class. According to the project requirements, all the methods in the `Book` class should be tested. Please see the example below as a suggestion on how to achieve this.

## Example code
```ruby
    it "The add_rental method returns a rental and adds its to the book" do
      book = Book.new("Title", "Author")
      person = Person.new(24, "Bob")

      rental = person.add_rental(book, "2021-05-04")

      expect(person.rentals).to eql [rental]
    end

```